doc/user_guide/mva.rst

```diff
@@ -104,7 +104,21 @@ the total variance specified, and the components above this value will be
 styled distinctly from the remaining components to show which are considered
 signal, as opposed to noise. Alternatively, by providing an integer value
 for ``threshold``, the line will be drawn at the specified component (see
-below). These options (together with many others), can be customized to
+below).  The number of significant components can be estimated and a vertical
+line drawn to represent this by specifying ``vline`` as ``True``. In this case,
+the elbow or knee is found in the variance plot by estimating the distance from
+each point in the variance plot to a line joining the first and last points of
+the plot.  As the index starts at zero, the number of significant PCA components
+is the elbow index position + 1.
+
+.. figure::  images/screeplot_elbow_method.png
+   :align:   center
+   :width:   500
+
+More details about the elbow or knee finding technique can be found in
```

Comment:
I think more details are needed here rather than just to the paper:

> In this case, the elbow or knee is found in the variance plot by estimating the distance from each point in the variance plot to a line joining the first and last points of the plot.

We should be clear about e.g. choosing between multiple candidates.

Reply:
Updated the description and commented on multiple candidates.